### ISSUE TITLE
AltServer 1.4.3 (Official) Mail Plugin still incompatible with Mail 14.0 on macOS 11.2

### ISSUE THREAD

**Issue by JDagsa** [rileytestut/AltStore#666]
I tried to enable the AltServer plugin in Mail 14.0 on macOS 11.2. It did not work.

<img width="263" alt="Screen Shot 2021-02-03 at 6 43 16 PM" src="https://user-images.githubusercontent.com/72021139/106735871-b892a580-664f-11eb-8db9-abe0965a7d46.png">

Tried re-installing, installing the public beta of 1.4.3 before updating to the official release of 1.4.3, still did not solve the issue.

Also tried the workaround used before AltServer 1.4.3, still did not work.
Any other workarounds or solutions?

Thank you.

MY MAC:
MacBook Air (M1, 2020)
macOS 11.2 (20D64)
Mail 14.0
AltServer 1.4.3 (50)

**Comment by JDagsa** [comment]
Holy smokes my issue no. is #666

**Comment by md-siam** [comment]
![Screen Shot 2021-02-12 at 11 11 38 PM](https://user-images.githubusercontent.com/21225918/107799719-02823680-6d88-11eb-8d67-57c2aa735557.png)
Same problem in macOS 11.1

**Comment by alberthenderson** [comment]
Issue still exist on MacOS BigSur 11.2.2 and AltServer 1.4.3

**Comment by md-siam** [comment]
@alberthenderson  did you use Apple TimeMachine to restore your mac?

**Comment by bakedpotato191** [comment]
I'm on 11.2.1 and still have the error. And, no, I did not use TimeMachine to restore my mac.

**Comment by Doregon** [comment]
@JDagsa 

AltStore works perfect on my Hackintosh (equivalent to a 2019 13" MacBook Pro, just with a touch screen) running the following OSes up to now:

- Catalina 10.15.6/10.15.7
- Big Sur Developer Beta 1-10/RC 2/11.0.1/11.1/11.2-11.2.2/11.3 beta 2

AltStore also works perfect on my 2012 15" MacBook Pro running the following:

- Mojave 10.14.6
- Catalina 10.15-10.15.7
- Big Sur 11.1/11.2-11.2.2 (patched with [this](https://github.com/barrykn/big-sur-micropatcher))

I wonder if it's an architectural issue (I don't exactly know, though--so don't count me on it) because of how it's working on Intel but not M1? @md-siam Do you have an M1 Mac as well?

**Comment by bakedpotato191** [comment]
> AltStore works perfect on my Hackintosh

Mine is also a Hackintosh Big Sur 11.2.2 on a ryzen cpu, and it hadn't worked in a while now, whether I reinstall mail plugin or not.

**Comment by Doregon** [comment]
@bakedpotato191 Alright, what I was saying was just a theory because I didn't know.

**Comment by Doregon** [comment]
@bakedpotato191 Aright, that's why it was just a theory and not a definitive statement.

**Comment by Doregon** [comment]
I don't know how much it will help but here's a copy of my Mail plugin, check if it works for you.

https://drive.google.com/file/d/1z-e_GZT_uTNfrOZS2pYFXOrTTFL4qnjZ/view?usp=sharing

**Comment by JDagsa** [comment]
> I wonder if it's an architectural issue (I don't exactly know, though--so don't count me on it) because of how it's working on Intel but not M1?

I have an M1 MacBook Air (MacBookAir10,1) Space Gray, base model.

**Comment by alberthenderson** [comment]
> @alberthenderson did you use Apple TimeMachine to restore your mac?

I'm not using Apple TimeMachine.
Currently I'm testing on Intel Macbook Pro and Apple M1 Macbook Pro, both MacOS 11.2.2, and AltServer 1.4.3

This issue already exist since early MacOS 11.1, my issue number [#612](https://github.com/rileytestut/AltStore/issues/612)

**Comment by Doregon** [comment]
@alberthenderson I don’t understand how it’s working for me then... I have SIP enabled, Gatekeeper is on and AltStore just asked if I wanted to update the Mail plugin from the one packaged with 1.4. Do we know where the actual plugin installs too? Maybe finding it and comparing what’s supposed to be in there could work.

**Comment by nyandrianinamamy** [comment]
Having this issue also, my only workaround is to use the windows version everytime I need to refresh apps

**Comment by Doregon** [comment]
Could someone get the value of the following arrays in `AltPlugin.mailbundle > Contents > Info.plist`?

`Supported11.0PluginCompatibilityUUIDs`
`Supported11.1PluginCompatibilityUUIDs`
`Supported11.2PluginCompatibilityUUIDs`
`Supported11.3PluginCompatibilityUUIDs`

**Comment by QuarterSwede** [comment]
They're all the same ID. This is after uninstalling the plugin and reinstalling it with Mail being shut down of course and confirming the plugin was deleted.

Supported11.0PluginCompatibilityUUIDs
D985F0E4-3BBC-4B95-BBA1-12056AC4A531
Supported11.1PluginCompatibilityUUIDs
D985F0E4-3BBC-4B95-BBA1-12056AC4A531
Supported11.2PluginCompatibilityUUIDs
D985F0E4-3BBC-4B95-BBA1-12056AC4A531
Supported11.3PluginCompatibilityUUIDs
D985F0E4-3BBC-4B95-BBA1-12056AC4A531

**Comment by Doregon** [comment]
Yeah, that's exactly what I had in my copy. I'm trying to figure out what's different between you and me (because AltStore works fine for me but not you).

**Comment by aBradleyUno** [comment]
Disable SIP... that worked for me

**Comment by QuarterSwede** [comment]
Disabling SIP is a security issue many aren’t willing to do.

**Comment by WrathofthePast** [comment]
Despite the v1.4.5 adding support for Mail 14.0 & MacOS 11.4+, I still get the incompatible with Mail 14.0 error on my MacBook Pro 13". Bizarrely, it works just fine on my MacBook Pro 16" after updating. I've tried a fresh install of AltServer, but to no avail. Any ideas?

Note: I did make sure the plugin updated too. I also tried replacing the info.plist file with the one from the working plugin on my 16".

**Comment by bakedpotato191** [comment]
> Despite the v1.4.5 adding support for Mail 14.0 & MacOS 11.4+, I still get the incompatible with Mail 14.0 error on my MacBook Pro

Same, despite the mail app throwing the alert, I am able to sideload the apps without any issues.
